addons/event/data/event_demo.xml

```diff
@@ -254,12 +262,14 @@
         <field name="name">Standard</field>
         <field name="event_id" ref="event.event_7"/>
         <field name="end_sale_datetime" eval="(DateTime.now() + timedelta(days=2)).strftime('%Y-%m-%d 15:00:00')"/>
+        <field name="sequence">1</field>
     </record>
     <record id="event_7_ticket_2" model="event.event.ticket">
         <field name="name">VIP</field>
         <field name="event_id" ref="event.event_7"/>
         <field name="end_sale_datetime" eval="(DateTime.now() + timedelta(days=2)).strftime('%Y-%m-%d 15:00:00')"/>
         <field name="seats_max">10</field>
+        <field name="sequence">2</field>
```

Comment:
following the _order = "sequence, event_id, price, name, id" on event ticket model,
ticket will then be ordered like this:
```
Event 1 | Ticket 1
Event 2 | Ticket 1
Event 3 | Ticket 1
...
Event 1 | Ticket 2
Event 2 | Ticket 2
...
```
Is it intended ? I thought @flch-odoo and you agreed on something like
_order = "event_id, sequence, price, name, id"
But in that case, I have another remark: Sequence is not done by event id, and if you use the handle widget to resequence tickets, let's say move the 6th ticket to the 2nd place, all the ticket that had 2 as sequence will be resequenced on a ascending order, cross event, and there would be no more duplicated sequences.

Or am I missing something ?

Reply:
This is because there is not place that we found that gathers cross-event tickets. It is either by event or as a bunch in the event.type model. So this should not happen in practice. However, if there was some let's say some reporting on all the tickets, that could actually happen.

However, now that I think about it, event_id would not be necessary if the situation does not present itself.

I've tried a bit locally and it seems only the tickets in the list are resequenced?
However if we prefer we can simply go event_id, sequence, price, name, id, and increment in the demo data? (no duplicates) ? What do you think @dbeguin ?